applications/FluidDynamicsApplication/custom_processes/two_fluids_inlet_process.cpp

```diff
@@ -0,0 +1,243 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Simon Wenczowski
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/openmp_utils.h"
+#include "utilities/variable_utils.h"
+
+// Application includes
+#include "two_fluids_inlet_process.h"
+#include "fluid_dynamics_application_variables.h"
+
+
+namespace Kratos
+{
+
+/* Public functions *******************************************************/
+
+TwoFluidsInletProcess::TwoFluidsInletProcess(
+    ModelPart& rModelPart,
+    Parameters& rParameters,
+    Process::Pointer p_dist_proc )
+    : Process(), mrInletModelPart(rModelPart) {
+
+    KRATOS_CHECK_VARIABLE_KEY( DISTANCE )
+    KRATOS_CHECK_VARIABLE_KEY( AUX_DISTANCE )
+    KRATOS_CHECK_VARIABLE_KEY( NODAL_AREA )
+
+    // checking all parameters that define the interface
+    Parameters default_parameters( R"(
+    {
+        "interface_normal"          : [0.0,1.0,0.0],
+        "point_on_interface"        : [0.0,0.25,0.0],
+        "inlet_transition_radius"   : 0.05
+    }  )" );
+
+    rParameters.ValidateAndAssignDefaults(default_parameters);
+
+    // finding the complete model of the inlet model part
+    ModelPart& rRootModelPart = mrInletModelPart.GetRootModelPart();
```

Comment:
r_root_model_part (naming convention again)

Reply:
Sorry for these unnecessary mistakes - I was too happy that something was working... Done